From 2ab8d619ca0f9817a68a2c906ead52fa0836f611 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:07:35 +0200
Subject: [PATCH 2/2] Merge android-9.0.0_r12

commit c38dc164452d6c6a03c9ce5adaa7f01b8cdc1fd4
Merge: a88551c0 342946fb
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jun 27 21:47:16 2018 +0000

    Merge "Snap for 4866033 from fbfde0bae50b9a6b01f56c80323ca22a6c4e14f0 to pi-dr1-release" into pi-dr1-release

commit a88551c041d3ddf4578e6708181806e345c2f087
Merge: f7164f3a fbfde0ba
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jun 27 21:40:47 2018 +0000

    Snap for 4866033 from fbfde0bae50b9a6b01f56c80323ca22a6c4e14f0 to pi-dr1-release

    Change-Id: If072e7d6d882dd6c03ac3b8cafbcef9de55b5b47

commit 342946fb94dd4aab1634ad05e19671d8fcaa4d22
Merge: f7164f3a fbfde0ba
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jun 27 21:39:14 2018 +0000

    Snap for 4866033 from fbfde0bae50b9a6b01f56c80323ca22a6c4e14f0 to pi-dr1-release

    Change-Id: I073d2d100cde8d0adf6cd9503c1e8e4d84c596d6

commit fbfde0bae50b9a6b01f56c80323ca22a6c4e14f0
Author: Jack Yu <jackyu@google.com>
Date:   Tue Jun 26 14:58:55 2018 -0700

    Fixed ETWS channel settings for KDDI

    Changed the ETWS channel settings based on the new
    KDDI open device requirement.

    Test: Manual
    Bug: 110233310
    Change-Id: Iedadb4d09245029e7146a96c4921fb1bc56867e1

commit f7164f3a5b203f912b2e2d94b8d74a56a763c50b
Merge: 3d863403 d22a15fa
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:03:26 2018 +0000

    Snap for 4859140 from d22a15fa4a33811fbe8a8a556744e2ce04abe712 to pi-dr1-release

    Change-Id: I1d1e37eba76462d720bc030ee28182df6c959670

commit d22a15fa4a33811fbe8a8a556744e2ce04abe712
Author: Bill Yi <byi@google.com>
Date:   Wed Jun 20 17:10:46 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I5df1b75b75267f1d8af4820ac48fc10f6c6da5d4

Change-Id: I1f177d439c45cb3a87ec2167b71ccd727865582e
---
 res/values-mcc440-mnc50/config.xml | 9 ++++++---
 res/values-mcc440-mnc51/config.xml | 9 ++++++---
 res/values-mcc440-mnc52/config.xml | 9 ++++++---
 res/values-uz/strings.xml          | 2 +-
 4 files changed, 19 insertions(+), 10 deletions(-)

diff --git a/res/values-mcc440-mnc50/config.xml b/res/values-mcc440-mnc50/config.xml
index 0db8f82f..7852691f 100644
--- a/res/values-mcc440-mnc50/config.xml
+++ b/res/values-mcc440-mnc50/config.xml
@@ -15,8 +15,11 @@
 -->
 
 <resources>
-    <string-array name="additional_cbs_channels_strings" translatable="false">
-        <item>0xAC00:type=test, emergency=true, scope=carrier</item>
-        <item>0x1105-0xAFFF:type=other, emergency=true, scope=carrier</item>
+    <!-- Based on KDDI open device requirement, only 4352 and 40963 should be enabled -->
+    <string-array name="etws_alerts_range_strings" translatable="false">
+        <!-- 0x1100 for earthquake -->
+        <item>0x1100:rat=gsm, emergency=true</item>
+        <!-- 0xA003 for Tsunami -->
+        <item>0xA003:rat=gsm, emergency=true, scope=carrier</item>
     </string-array>
 </resources>
diff --git a/res/values-mcc440-mnc51/config.xml b/res/values-mcc440-mnc51/config.xml
index 0db8f82f..7852691f 100644
--- a/res/values-mcc440-mnc51/config.xml
+++ b/res/values-mcc440-mnc51/config.xml
@@ -15,8 +15,11 @@
 -->
 
 <resources>
-    <string-array name="additional_cbs_channels_strings" translatable="false">
-        <item>0xAC00:type=test, emergency=true, scope=carrier</item>
-        <item>0x1105-0xAFFF:type=other, emergency=true, scope=carrier</item>
+    <!-- Based on KDDI open device requirement, only 4352 and 40963 should be enabled -->
+    <string-array name="etws_alerts_range_strings" translatable="false">
+        <!-- 0x1100 for earthquake -->
+        <item>0x1100:rat=gsm, emergency=true</item>
+        <!-- 0xA003 for Tsunami -->
+        <item>0xA003:rat=gsm, emergency=true, scope=carrier</item>
     </string-array>
 </resources>
diff --git a/res/values-mcc440-mnc52/config.xml b/res/values-mcc440-mnc52/config.xml
index 0db8f82f..7852691f 100644
--- a/res/values-mcc440-mnc52/config.xml
+++ b/res/values-mcc440-mnc52/config.xml
@@ -15,8 +15,11 @@
 -->
 
 <resources>
-    <string-array name="additional_cbs_channels_strings" translatable="false">
-        <item>0xAC00:type=test, emergency=true, scope=carrier</item>
-        <item>0x1105-0xAFFF:type=other, emergency=true, scope=carrier</item>
+    <!-- Based on KDDI open device requirement, only 4352 and 40963 should be enabled -->
+    <string-array name="etws_alerts_range_strings" translatable="false">
+        <!-- 0x1100 for earthquake -->
+        <item>0x1100:rat=gsm, emergency=true</item>
+        <!-- 0xA003 for Tsunami -->
+        <item>0xA003:rat=gsm, emergency=true, scope=carrier</item>
     </string-array>
 </resources>
diff --git a/res/values-uz/strings.xml b/res/values-uz/strings.xml
index e61be35a..d35aca06 100644
--- a/res/values-uz/strings.xml
+++ b/res/values-uz/strings.xml
@@ -23,7 +23,7 @@
     <string name="no_cell_broadcasts" msgid="5409324337492758562">"Ayni paytda siz joylashgan hudud uchun hech qanday faol ogohlantirish yo‘q. Ogohlantirish parametrlarini Sozlamalar menyusidan o‘zgartirish mumkin."</string>
     <string name="menu_delete_all" msgid="3940997343921149800">"Tarqatma xabarlarni o‘chirish"</string>
     <string name="message_options" msgid="3178489901903589574">"Xabar sozlamalari"</string>
-    <string name="menu_view_details" msgid="1040989019045280975">"Tafsilotlarni ko‘rish"</string>
+    <string name="menu_view_details" msgid="1040989019045280975">"Tafsilotlar"</string>
     <string name="menu_delete" msgid="128380070910799366">"Tarqatma xabarni o‘chirish"</string>
     <string name="view_details_title" msgid="1780427629491781473">"Ogohlantirish tafsilotlari"</string>
     <string name="confirm_delete_broadcast" msgid="2540199303730232322">"Ushbu tarqatma xabar o‘chirib tashlansinmi?"</string>
-- 
2.17.1

